app/system/survey/js/prompts.js

```diff
@@ -2131,8 +2136,22 @@ promptTypes.date = promptTypes.datetime.extend({
     timeFormat: "MM/DD/YYYY",
     timeTemplate: "YYYY / MM / DD"
 });
+promptTypes.dateshort = promptTypes.datetime.extend({
+    type: "dateShort",
+    showTime: false,
+    timeFormat: "MM/DD/YYYY",
```

Comment:
Why is timeFormat not in the same format with YYYY then MM then DD as the other things. Helps with internationalization of dates.

Reply:
The time format and time template are used for *display*. I left those to be consistent. I agree that I would personally change their order, but I think that is probably a different PR where we're intentionally deciding to reorder our date displays (it would be a pretty quick and easy PR). 

The `dbDateFormat` is what is used to *store* the date, which mirrors the ISO standard and is similar to how the current dates are stored (but without the time, obviously). 

So the intention was to add the desired functionality without changing any other functionality.